### PR TITLE
Upgrade to v1.4.2

### PR DESCRIPTION
**Pull request by notmahsa**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
 FROM debian
 
-RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
-RUN apt-get update
-RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install postgresql
+RUN apt update
+RUN DEBIAN_FRONTEND="noninteractive" apt -y install tzdata
+RUN DEBIAN_FRONTEND="noninteractive" apt -y install postgresql
```
Comment (heyollie) [inline Dockerfile:5]: May be considered best practices to merge these `RUN` statements:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -2,29 +2,61 @@
 set -e
```
Comment (heyollie) [inline docker-entrypoint.sh:2]: `set -euo pipefail` for saftey https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425

File: docker-entrypoint.sh [hunk 3]
```diff
@@ -2,29 +2,61 @@
...
+    -domain "$GC_DOMAIN" \
+    -email "$GC_USER_EMAIL" \
+    -password "$GC_PASSWORD" \
+    -db "$DB_CONNECTION_STRING"
+}
+
+{
+  migrate_db
+} || {
+  echo 'Failed to run migrations'
```
Comment (heyollie) [inline docker-entrypoint.sh:27]: We want to exit after these failures right? `set -o pipefail` at the top should do that I believe

**Review by heyollie — APPROVED** [review]
Looks good to me, added some minor comments with learnings I had during my OSS.

File: docker-entrypoint.sh (hunk 3 above)
Comment (notmahsa) [reply docker-entrypoint.sh:27]: We don't want to exit after these two failures, these only show we have a site already created and no migrations to run

Comment (heyollie) [reply docker-entrypoint.sh:27]: got it! also I was wrong about pipefail, it shouldn't affect this

File: docker-entrypoint.sh [hunk 4]
```diff
@@ -2,29 +2,61 @@
 set -e
 
 export GC_DOMAIN="${RENDER_EXTERNAL_HOSTNAME}"
+export DB_HOSTNAME=${DB_HOSTNAME:-'oregon-postgres.render.com'}
```
Comment (jakemalachowski) [inline docker-entrypoint.sh:5]: Is `DB_HOSTNAME` expected to be set? I'm not seeing it set anywhere, but I may be missing it. If it's not set, wouldn't this cause EU deployments to fail?

**Review by jakemalachowski — COMMENTED** [review]
Looks good! Just the one question that might need to be addressed

File: docker-entrypoint.sh (hunk 4 above)
Comment (notmahsa) [reply docker-entrypoint.sh:5]: It will be taken from the service env vars (user entry)

**Review by jakemalachowski — APPROVED** [review]

**Pull request merged**